firmware/library/L2_Utilities/util.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+
+namespace Utility
+{
+static uint32_t Random(uint32_t min, uint32_t max);
+static void SetRandomSeed();
```

Comment:
I think that `SetRandomSeed()` should allow the developer specify a random seed input. 
You can have it default to 0xFFFFFFFF, in that case, you can use the <ctime> library in the implementation.